tests/includes/charmcraft.sh

```diff
@@ -0,0 +1,12 @@
+# pack_charm uses charmcraft to pack the local charm at the given directory,
+# and returns a path to the packed charm which can be supplied to juju deploy.
+#
+# Example usage:
+#    juju deploy $(pack_charm ./testcharms/charms/lxd-profile)
+pack_charm() {
+	local CHARM_DIR=$1
+	CHARM_NAME=$(basename "$CHARM_DIR")
+
+	charmcraft pack -p "$CHARM_DIR"
```

Comment:
If you want to see the messages and we want to ensure it fails correctly, even when inline you **could** do the following.

```suggestion
    set -eux
	charmcraft pack -p "$CHARM_DIR"
	set_verbosity
```